docs/digipres.rst

```diff
@@ -0,0 +1,164 @@
+About digital preservation
+==========================
+
+At Artefactual, we work in the context of archival standards.
+This context informs the decisions we make for the software we
+develop and maintain.
+
+The resources listed here can be specific to our own context, since
+there are a diverse, broad range of archival traditions in the
+world. We believe that the list presented here is a good first step
+to get started in the digital preservation world. This list has been
+curated for beginners.
+
+Digital preservation is for everyone
+------------------------------------
+
+This document is on resources about digital preservation and archives
+in general. The goal is to help someone new to this world get
+familiar with the concepts and ideas, not to replace a formal
+education on the topic.
+This is not meant to be exhaustive.
+
+What is digital preservation?
+-----------------------------
+
+Digital preservation is the set of actions, standards and processes
+involved in making digital or digitalized content available in the
+future.
+
+Digital preservation might involve standard IT storage solutios,
+with a layer of additional needs to be met.
+
+Getting started
+---------------
+
+The `Novice to Know-How: Digital Preservation for Beginners`_
+from the `Digital Preservation Coalition`_ is one of the
+most thorough yet concise training on the topic.
+It covers file formats, bitstreams, workflows for preservation,
+and how to give access to preserved digital content.
+It will help you understand the risks associated with preserving
+digital content in the long term, from the material point of view.
+
+Note that there is also a `DPC YouTube channel`_.
+
+Standards
+---------
+
+Our work is informed both by archival content standards and by
+metadata standards.
+
+Metadata standards
+``````````````````
+
+Software developed at Artefactual is informed by the digital
+preservation standards, such as `PREMIS`_ and
+`METS`_. Both these websites contain learning materials,
+such as this `PREMIS list of tutorials`_
+as well as `suggested readings for METS`_.
+
+Archivematica documentation has pages on
+`how PREMIS is supported in the software`_,
+and similarly for `METS in Archivematica`_.
+
+Archival content standards
+``````````````````````````
+
+Some of the archival contents standards we are interested in
+includes the Rules for Archival Description, or `RAD`_, the
+`DACS`_ and `ISAD-G`_. Other standards might be
+of interest which are not listed here.
+
+Technical challenges
+--------------------
+
+If you are curious about the technical challenges of preserving data
+and its metadata for the long term, consider reading about the
+`challenges faced by the CERN Digital Memory Platform`_,
+such as scaling, hardware failures, losing data because of software
+errors, and other ways that digital preservation can fail to happen.
+This thesis make use of Archivematica.
+
+Learning to code
+----------------
+
```

Comment:
Would the audience for this not already know how to code?

Reply:
I guess it could be rephrased... let me see.

Reply:
I added suggestions for lines 83 and 86 - let me know!